src/marqo/tensor_search/tensor_search.py

```diff
@@ -758,7 +759,8 @@ def _add_documents_structured(config: Config, add_docs_params: AddDocsParams, ma
                                      f"Is this a new tensor field?")
 
                 if len(chunks) == 0:  # Not using existing tensors or didn't find it
-                    if isinstance(field_content, (str, Image.Image)):
+                    is_image_pointer_field = marqo_field.type == FieldType.ImagePointer
+                    if isinstance(field_content, str):
```

Comment:
Why were we checking for Image.Image before?

Reply:
I think we wanted to support indexing images as bytes but the feature was parked